samples/bluetooth/mesh/sensor_server/README.rst

```diff
@@ -0,0 +1,186 @@
+
+.. include:: /includes/build_and_run.txt
+
+.. _bluetooth_mesh_sensor_server_testing:
+
+Testing
+=======
+
+.. note::
+   The Mesh sensor server sample cannot demonstrate any functionality on its own, and needs a device with the :ref:`bluetooth_mesh_sensor_client` sample running in the same Mesh network.
+   Before testing the Mesh sensor server, go through the Mesh sensor client's :ref:`testing guide <bluetooth_mesh_sensor_client_testing>` with a different board.
+
+After programming the sample to your board, you can test it by using a smartphone with Nordic Semiconductor’s nRF Mesh app installed.
+Testing consists of provisioning the device and configuring it for communication with the mesh models.
+
+Provisioning the device
+-----------------------
+
+The provisioning assigns an address range to the device, and adds it to the mesh network.
+Complete the following steps in the nRF Mesh app:
+
+1. Tap :guilabel:`Add node` to start scanning for unprovisioned mesh devices.
+#. Select the :guilabel:`Mesh Sensor Server` device to connect to it.
+#. Tap :guilabel:`Identify` and then :guilabel:`Provision` to provision the device.
+#. When prompted, select an OOB method and follow the instructions in the app.
+
+Once the provisioning is complete, the app returns to the Network screen.
+
+.. _bluetooth_mesh_sensor_server_conf_models:
+
+Configuring models
+------------------
+
+Complete the following steps in the nRF Mesh app to configure models:
+
+1. On the Network screen, tap the :guilabel:`Mesh Sensor Server` node.
+   Basic information about the mesh node and its configuration is displayed.
+#. In the Mesh node view, expand the element.
+   It contains the list of models in the first and only element of the node.
+#. Tap :guilabel:`Sensor Server` to see the model's configuration.
+#. Bind the model to application keys to make it open for communication:
+
+   a. Tap :guilabel:`BIND KEY` at the top of the screen.
+   #. Select :guilabel:`Application Key 1` from the list.
+
+#. Set the publishing parameters:
+
+   a. Tap :guilabel:`SET PUBLICATION`.
+   #. Tap :guilabel:`Publish Address`.
+   #. Select :guilabel:`Groups` from the drop-down menu.
+   #. Select an existing group or create a new one.
+
+      .. note::
+            The sensor client must subscribe to the same group.
+
+   #. Tap :guilabel:`OK`.
+   #. Select a publishing period by using the :guilabel:`Interval` slider.
+   #. Set the Retransmit Count to zero (:guilabel:`Disabled`) to avoid duplicate logging in the :ref:`sensor client's <bt_mesh_sensor_cli_readme>` UART terminal.
+   #. Tap :guilabel:`Check` at the bottom right corner.
+
+#. Set subscription parameters:
+
+   a. Tap :guilabel:`SUBSCRIBE`.
+   #. Select an existing group or create a new one.
+
+      .. note::
+            The sensor client must publish to the same group.
+
+   #. Tap :guilabel:`OK`.
+
+#. Tap the back arrow at the top left corner.
```

Comment:
See my comment to the client.

Reply:
This is a "back button" to get back to the previous menu. If it is pressed twice you get back to the "main menu". Maybe it is better to write something like this:
"The sensor server model is now configured and should be able to send data to the client. Press the back arrow (top left corner) twice to get back to the main screen. "